tasks/mongodb.yml

```diff
@@ -22,6 +22,16 @@
   notify:
     - restart mongo
 
+- name: set tmpfs size for mongodb
+  become: true
+  lineinfile:
+    path: /usr/share/meteor/bundle/mongod_start_pre.sh
+    regexp: "^  mount -t tmpfs"
+    line: "  mount -t tmpfs -o size={{ bbb_mongodb_tmpfs_size }} tmpfs /mnt/mongo-ramdisk"
+  notify:
+    - stop mongo
+    - restart mongo
+
```

Comment:
maybe template this script?
Or is that too much?

Reply:
I would go for a template, too. Then you can easily see the context, just by looking at the role.
Why do you stop and then restart mongo?